pmoves/waitlist/styles.css

```
@@ -0,0 +1,407 @@
+}
+
+/* ─── Screen Reader Only ─────────────────── */
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)